README.md

```diff
@@ -1,84 +1,9 @@
-```
-
-## Install KubeStellar Core chart
-
-A KubeStellar Core installation compatible with the common setup suitable for Common Setup described in the [examples](examples.md) could be achieved with the following command:
-
-```shell
-helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart --version $KUBESTELLAR_VERSION \
-  --set-json='ITSes=[{"name":"its1"}]' \
-  --set-json='WDSes=[{"name":"wds1"}]'
-```
-
-Remember to include the option `--set "kubeflex-operator.isOpenShift=true"`, when deploying in an OpenShift cluster.
-
-## Kubeconfig contexts for Control Planes
-
-After the KubeStellar Core has fully deployed, before changing the kubeconfig context, `kflex` CLI can be used to retrieve the Kubernetes client configurations for all the KubeFlex Control Planes and store them as contexts of the current kubeconfig file:
-
-```shell
-kubectl config delete-context its1 || true
-kflex ctx its1
-kubectl config delete-context wds1 || true
-kflex ctx wds1
-kflex ctx # switch back to the initial context
-```
-
-Afterwards the content of a Control Plane `<cpname>` can be accessed by specifing its context:
-
-```shell
-kubectl --context <cpname> ...
-```
-
-## (optional) Check relevant deployments and statefulsets running in the hosting cluster
-
-Expect to see the `kubestellar-controller-manager` in the `wds1-system` namespace and the
-statefulset `vcluster` in the `its1-system` namespace, both fully ready.
-
-```shell
-kubectl get deployments,statefulsets --all-namespaces
-```
-
-The output should look something like the following:
-
-```shell
-NAMESPACE            NAME                                             READY   UP-TO-DATE   AVAILABLE   AGE
-ingress-nginx        deployment.apps/ingress-nginx-controller         1/1     1            1           22h
-kube-system          deployment.apps/coredns                          2/2     2            2           22h
-kubeflex-system      deployment.apps/kubeflex-controller-manager      1/1     1            1           22h
-local-path-storage   deployment.apps/local-path-provisioner           1/1     1            1           22h
-wds1-system          deployment.apps/kube-apiserver                   1/1     1            1           22m
-wds1-system          deployment.apps/kube-controller-manager          1/1     1            1           22m
-wds1-system          deployment.apps/kubestellar-controller-manager   1/1     1            1           21m
-wds1-system          deployment.apps/transport-controller             1/1     1            1           21m
-
-NAMESPACE         NAME                                   READY   AGE
-its1-system       statefulset.apps/vcluster              1/1     11h
-kubeflex-system   statefulset.apps/postgres-postgresql   1/1     22h
-```
+Note: this repository was used for Helm charts for older releases of KubeStellar.
```

Comment:
I'd expand this with something like  _... older releases of KubeStellar; be sure to visit kubestellar.io and/or github.com/kubestellar/kubestellar for the newer releases_
to really drive the point home.

Reply:
But isn't that very redundant with the previous paragraph?

Reply:
It is redundant, but I think stressing that these are possibly obsolete files might be worth saying twice.

Reply:
done